examples/scalajs-react/generated/todomvc-launcher.js

```diff
@@ -0,0 +1,2 @@
+((typeof global === "object" && global &&
+         global["Object"] === Object) ? global : this)["todomvc"]["Main"]().main();
```

Comment:
Yeah i did first rewrite it like we agreed. 

The reason why it looks like this again now, is that it is now generated (look at the path) by the sbt scala.js plugin - in Build.sbt:
```diff
+      /* create javascript launcher. Searches for an object extends JSApp */
+      persistLauncher                := true,
```

I guess we can agree that generated code is not subject to the style guide? :)